site-manage/datemanager/tool/src/main/webapp/WEB-INF/templates/confirm_import.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
+	"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
+	<head>
+		<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+	</head>
+	<body class="portalBody Mrphs-portalBody Mrphs-sakai-datemanager-tool">
+		<link media="all" href="/library/skin/tool_base.css" rel="stylesheet" type="text/css" />
+		<script src="/library/js/headscripts.js" type="text/javascript"></script>
+		<script type="text/javascript" src="/library/js/lang-datepicker/lang-datepicker.js"></script>
+		<script th:src="@{/js/initDatePicker.js}"></script>
+		<meta http-equiv="Content-Style-Type" content="text/css" />
+		<div class="page-header">
+			<h1 th:text="#{page.confirm.import}">Confirmation: Import dates</h1>
+		</div>
+		<div class="sak-banner-error" th:if="${errors != null}">
+			<table>
+				<thead>
+					<tr>
+						<td th:text="#{errors.instruction}" colspan="2">You cannot do that changes beacuse: </td>
+					</tr>
+				</thead>
+				<tbody th:each="error : ${errors}">
+					<tr class="h4">
+						<th th:text="${error.get(0).getErrors().get(0).toolTitle}">Tool title</th>
+					</tr>
+					<tr>
+						<td th:text="#{template.error(${error.get(1)}, ${error.get(2)})}" class="pr-20">You cannot do that changes beacuse: </td>
+						<td th:each="errorValidation : ${error.get(0).getErrors()}" th:text="#{template.error(${errorValidation.msg}, ${errorValidation.field})}">You cannot do that changes beacuse: </td>
```

Comment:
```suggestion
						<td th:text="#{template.error(${error.get(1)}, ${error.get(2)})}" class="pr-20">You cannot do those changes because: </td>
						<td th:each="errorValidation : ${error.get(0).getErrors()}" th:text="#{template.error(${errorValidation.msg}, ${errorValidation.field})}">You cannot do those changes because: </td>
```

Even if they're overriden by the properties, these texts have spelling errors.